packages/runtime/html_rewriter.patch

```diff
@@ -0,0 +1,4 @@
+56c56
+<       fetch(new URL("./vendor/html_rewriter_bg.wasm", import.meta.url).href)
+---
```

Comment:
This looks like we're replacing the from-disk fetch with a HTTP request at runtime here. This is required because `deno vendor` doesn't download the WASM onto disk (for whatever reason),  so fetching this from disk would fail. This was already the case before this PR, but before this file was imported via `https://`, meaning that `import.meta.url` was something like `https://deno.land/x/...` - now that we're accessing this file from disk, it's more like `file://.netlify/edge-functions/vendor...`. Before, with the HTTPS address, `fetch` would fall back to an HTTP request anyways.

So TL;DR: This looks like we're replacing a solid disk-read with a brittle network fetch. In reality, it's always been a brittle network fetch, so this is fine!

Reply:
Can we download that wasm file ourselves and manually vendor it? i.e. manually pull from https://github.com/worker-tools/html-rewriter/blob/v0.1.0-pre.17/vendor/html_rewriter_bg.wasm and avoid doing a  patch (it's bit icky to me, but manual download is icky as well, so I guess - pick your poison). This way at least we would avoid this brittle network fetch?

Reply:
Tried that, but the problem is *not only* that `deno vendor` doesn't download WASM files - Deno's eszip tool (that we use for bundling) also isn't able to detect the import, so it doesn't bundle the file. This will probably change once https://github.com/tc39/proposal-source-phase-imports lands, but until then the HTTP fetch is the only thing we can do :/